### PR TITLE
Use sqlite for bundle

### PR DESCRIPTION
**Pull request by namse**
Use sqlite for bundle will make a big performance improvement for windows(https://www.sqlite.org/fasterthanfs.html), and also I'd like to change bundle system in wasi more simple with sqlite.

Windows not tested.
Wasi tested.

Breaking changes: no `bundle:` for bundle url.
```rust
// before
let buffer = namui::system::file::bundle::read("bundle:resources/text.txt");

// after
let buffer = namui::system::file::bundle::read("resources/text.txt");
```

### CODE REVIEW

**Comment by namse** [comment]
@bigfoodK Could you make it work on windows?

**Review by bigfoodK — APPROVED** [review]

File: namui/namui-cli/src/services/resource_collect_service.rs [hunk 1]
```diff
@@ -149,33 +149,46 @@ impl CollectOperation {
...
-    }
+    let dest_dir_path = dest_dir_path.as_ref();
+    debug_println!(
+        "resource_collect_service: copy {:?} -> {:?}/{:?}",
+        &from,
+        &dest_dir_path,
+        &from.file_name().unwrap()
+    );
+    ensure_dir(dest_dir_path)?;
+    assert!(dest_dir_path.is_file());
```
Comment (bigfoodK) [inline namui/namui-cli/src/services/resource_collect_service.rs:174]: You mean `assert!(dest_dir_path.is_dir());`?

Comment (namse) [reply namui/namui-cli/src/services/resource_collect_service.rs:174]: oh, `assert!(from.is_file())`;

Comment (namse) [reply namui/namui-cli/src/services/resource_collect_service.rs:174]:
```suggestion
    assert!(from.is_file());
```

**Pull request merged**